lib/commons/aria/index.js

```diff
@@ -78,16 +85,20 @@ lookupTable.attributes = {
 	},
 	'aria-invalid': {
 		type: 'nmtoken',
+		allowEmpty: true,
 		values: ['true', 'false', 'spelling', 'grammar']
 	},
 	'aria-keyshortcuts': {
-		type: 'string'
+		type: 'string',
+		allowEmpty: true
 	},
 	'aria-label': {
-		type: 'string'
+		type: 'string',
+		allowEmpty: true
```

Comment:
Do we have a test to make sure that an empty aria-label will not pass an input with `aria-label=""`?

Reply:
`aria-label` has always been allowed empty. See: https://github.com/dequelabs/axe-core/pull/1154/files#diff-0dbb83fa6e11da52031f948e3f85a3ecL105

Rules that require elements to have an accessible name don't allow empty or whitespace values to pass.